src/priv-csrs.adoc

```diff
@@ -576,7 +576,7 @@ Machine trap cause. +
 Machine bad address or instruction. +
 Machine interrupt pending. +
 Machine trap instruction (transformed). +
-Machine bad guest physical address.
+Machine auxilliary trap information.
```

Comment:
"Machine second trap value" is better. 
Maybe could also fix up other labels to match the description in the Machine, Supervisor, and H extension ISA section  (e.g., bad address or instruction -> Machine Trap Value"

Reply:
Will update this to match the names used in the other section headings.